cmd/osm-bootstrap/crds/config_meshconfig.yaml

```diff
@@ -99,12 +99,12 @@ spec:
                         - TLSv1_3
                       default: TLSv1_2
                     tlsMaxProtocolVersion:
-                      description: The maximum TLS protocol version that the sidecar supports. Valid TLS protocol versions are TLS_AUTO, TLSv1_0, TLSv1_1, TLSv1_2 and TLSv1_3.
+                      description: The maximum TLS protocol version that the sidecar supports. Valid TLS protocol versions are TLS_AUTO, TLSv1_2 and TLSv1_3.
                       type: string
                       enum:
                         - TLS_AUTO
-                        - TLSv1_0
-                        - TLSv1_1
+                        - TLSv1_0 # Deprecated 
```

Comment:
This YAML comment won't be displayed to the user; could you modify the description above?